### PR TITLE
Bug fix - data not being cleared when going from a 'Yes' option to a 'No' option

### PR DESCRIPTION
**Pull request by gclssvglx**
What
----

When going from a 'Yes' option to a 'No' option on some pages, the old values and data was not being cleared correctly.

This change resolves the issue.

Links
-----

[Trello cards](https://trello.com/c/eGsOEifg/488-update-check-your-answers-page)

### CODE REVIEW

**Review by reggieb — DISMISSED** [review]
LGTM

**Review by leenagupte — COMMENTED** [review]
The code looks good! 
Are there any tests that can be added?

**Comment by gclssvglx** [comment]
Tests added.

File: spec/controllers/coronavirus_form/offer_care_controller_spec.rb [hunk 1]
```diff
@@ -55,11 +55,17 @@
       expect(response).to redirect_to(offer_other_support_path)
     end
 
-    it "clears previously entered care cost for a 'No' response" do
+    it "clears previously entered answers given a 'No' response" do
+      session[:offer_care_type] = I18n.t("coronavirus_form.questions.offer_care_qualifications.offer_care_type.options").map { |_, item| item[:label] }.sample
```
Comment (reggieb) [inline spec/controllers/coronavirus_form/offer_care_controller_spec.rb:59]: I'd suggest a better pattern to `.map { |_, item| .... }` is `.values.map { |item| }`

**Review by reggieb — APPROVED** [review]
Thank you for adding the specs.

File: spec/controllers/coronavirus_form/offer_care_controller_spec.rb (hunk 1 above)
Comment (gclssvglx) [reply spec/controllers/coronavirus_form/offer_care_controller_spec.rb:59]: Thanks @reggieb - good spot 👍  

I agree, just mapping the values makes more sense than defining a useless and subsequently ignored variable.  However, as the `.map { |_, item| .... }` pattern is used throughout the form, so I'll go with the `prior art` approach.

**Pull request merged**